### DISCUSSION TITLE
Beta users for Sign in with Google

### DISCUSSION THREAD

**Discussion by morganney** [morganney/busmap#164] [Announcements]
Busmap is now in beta testing at [try.busmap.online](https://try.busmap.online). If you want to be a test user for [Sign in with Google](https://developers.google.com/identity/gsi/web/guides/overview) button, and have a valid gmail address, let me know here and I'll provide a way for you to contact me with your gmail address.

**Comment by rajeshrah22** [comment]
Hi Morgan,

Firstly I want to say, your restbus project is pretty cool since I used the nextBus xml feed in a side project to learn how HTTP, APIs work. What I built is very simply, it just lists agencies, routes, stops and plots them on the map. I didn't think it can be of much use since people use google maps anyways to look for local bus routes(at least in my experience).

UI sucks but you can check it out here: 
http://3.80.204.166/BusApp/

currently working on a more modern looking UI using react.js.

I just find it funny that other people also have used this API to make things.

What's your reason for using the nextBus API?

**Reply by morganney** [reply]
Hi Rahul,

Thanks, I developed restbus over a decade ago to get some more experience with node.

Your project using the nextbus XML feed is cool, I can see how it could be helpful to people wanting to know the path and stops of their most used transit agencies. I went back and forth between google maps and leaflet, but didn't want to have to worry about incurring any costs so went with leaflet.

I used the NextBus API because I worked there for a coupe of years before they were acquired by Cubic. I actually started looking for other free transit agency API's because I've noticed the NextBus (umoiq) XML feed has started dropping consistent support for some popular transit agencies, like San Francisco Muni  🤷‍♂️ . Currently trying to get an API key for [Swiftly](https://www.goswift.ly/platform).

Did you want me to add your gmail address as a test user? It simply allows you to sign in where your configured UI settings are saved and you can favorite more than three stops. Other than that, the UI behaves exactly the save for unauthenticated users.

**Reply by rajeshrah22** [reply]
Yeah sure, you can put this email for test user : mypigsflyhifi@gmail.com

**Reply by morganney** [reply]
Ok, I've added you. Please let me know if you encounter any issues.
